plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAnalyze.java

```diff
@@ -696,9 +696,14 @@ public void testIncrementalStatisticsUpdateOnInsert()
     }
 
     private static Session disableStatisticsCollectionOnWrite(Session session)
+    {
+        return setStatisticsCollectionOnWrite(session, false);
+    }
+
+    private static Session setStatisticsCollectionOnWrite(Session session, boolean value)
```

Comment:
in https://github.com/trinodb/trino/pull/15441 I call similar method `withStatsOnWrite`